cell.go

```diff
@@ -142,6 +142,18 @@ func (f *File) SetCellBool(sheet, axis string, value bool) {
 	}
 }
 
+// SetCellFloat sets a floating point value into a cell. The prec parameter
+// specifies how many places after the decimal will be shown while -1
+// is a special value that will use as many decimal places as necessary to
+// represent the number
+func (f *File) SetCellFloat(sheet, axis string, value float64, prec int) {
+	xlsx := f.workSheetReader(sheet)
+	cellData, col, _ := f.prepareCell(xlsx, sheet, axis)
+	cellData.S = f.prepareCellStyle(xlsx, col, cellData.S)
+	cellData.T = ""
+	cellData.V = strconv.FormatFloat(value, 'f', prec, 64)
```

Comment:
we can't convert float32 type numbers by 64-bit size bits for uniform processing.

Reply:
Yeah I should have read the docs for `FormatFloat` more carefully. I'll add another parameter to SetCellFloat for this.